msgpack_test.go

```diff
@@ -1,4 +1,4 @@
-package msgpack_test
+package msgpack
```

Comment:
I thought you achieved that with build tag - `go build -tags=dev`. AFAIK this change is not needed.